Python/initconfig.c

```diff
@@ -2085,7 +2106,7 @@ config_read(PyConfig *config, int compute_path_config)
     /* -X options */
     const wchar_t* option = _Py_check_xoptions(&config->xoptions, known_xoptions);
     if (option != NULL) {
-        return PyStatus_Error("Unknown value for option -X");
+        return PyStatus_Error("Unknown value for option -X (see --help-xoptions)");
     }
```

Comment:
Turns out that showing the invalid value in this error message is not trivial: real C programmers had issue with it too before https://github.com/python/cpython/pull/28823